tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -956,7 +956,12 @@ def __init__(self, logger, browser, server_config, timeout_multiplier=1,
         TestharnessExecutor.__init__(self, logger, browser, server_config,
                                      timeout_multiplier=timeout_multiplier,
                                      debug_info=debug_info)
-        self.protocol = self.protocol_cls(self, browser, capabilities)
+
+        self.require_webdriver_bidi = kwargs.get("browser_settings", {}).get(
+            "require_webdriver_bidi", False)
```

Comment:
We should just add `browser_settings` to the list of arguments and not have to touch `kwargs`; it's always passed by the runner. c.f. https://github.com/web-platform-tests/wpt/blob/0ceef6fec280d11d8dca23d32cedcb06ba4c85da/tools/wptrunner/wptrunner/testrunner.py#L651